cmd/kubectl-gadget/traceloop.go

```diff
@@ -70,131 +63,56 @@ var traceloopShowCmd = &cobra.Command{
 	RunE:  runTraceloopShow,
 }
 
-var traceloopPodCmd = &cobra.Command{
-	Use:   "pod",
-	Short: "show the traces in one pod",
-	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) != 3 {
-			return commonutils.WrapInErrMissingArgs("<namespace>, <podname> and <idx>")
-		}
-		return nil
-	},
-	RunE: runTraceloopPod,
-}
-
-var traceloopCloseCmd = &cobra.Command{
-	Use:   "close",
-	Short: "close one trace",
-	RunE:  runTraceloopClose,
-}
-
-var (
-	optionListFull          bool
-	optionListAllNamespaces bool
-	optionListNoHeaders     bool
-)
-
 func init() {
 	rootCmd.AddCommand(traceloopCmd)
+	utils.AddCommonFlags(traceloopCmd, &params)
+
 	traceloopCmd.AddCommand(traceloopStartCmd)
 	traceloopCmd.AddCommand(traceloopStopCmd)
 	traceloopCmd.AddCommand(traceloopListCmd)
 	traceloopCmd.AddCommand(traceloopShowCmd)
-	traceloopCmd.AddCommand(traceloopPodCmd)
-	traceloopCmd.AddCommand(traceloopCloseCmd)
```

Comment:
* Since the "close" command was removed, the user cannot removes old logs from old pods. The memory consumption will grow indefinitely when pods are created and deleted.

* We should also re-add something similar to the "pod" command to make the CLI more human friendly.

This can be fixed in a future PR.